src/iqm/qiskit_iqm/iqm_qiskit_plugin.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Qiskit on IQM developers
+"""Transpiler plugins for IQM N-star architecture"""
+
+from typing import Optional
+
+from qiskit.transpiler.passmanager import PassManager
+from qiskit.transpiler.passmanager_config import PassManagerConfig
+from qiskit.transpiler.preset_passmanagers.plugin import PassManagerStagePlugin
+
+from iqm.qiskit_iqm.iqm_naive_move_pass import build_IQM_star_pass
+
+
+class TranslateIQMPlugin(PassManagerStagePlugin):
```

Comment:
I would change it to a more specific name, specifying it is specifically for n-star architecture.

Reply:
Yes, the plugin name can be changed to avoid confusion with other device structures

Reply:
Plugin removed from the published package / PR.